### PR TITLE
Modify test/e2e/fishapp to handle load testing

### PR DESCRIPTION
**Pull request by vishalathreya**
*Issue #, if available:*

*Description of changes:*
Existing fishapp e2e test uses static resources (fixed VN, VS, etc counts). PR adds feature to read that from config file.
PR also handles deployment of Fortio (open source load generator) components and triggering load test from provided script.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: test/e2e/fishapp/load/load_stack_test.go [hunk 1]
```diff
@@ -0,0 +1,279 @@
...
+
+			portForwardCmd := deployFortio(ctx, f, stacksPendingCleanUp)
+			defer func() {
+				fmt.Println("killing Fortio port-forward process")
+				if portForwardKillErr := portForwardCmd.Process.Kill(); portForwardKillErr != nil {
+					fmt.Println("failed to kill process. Error -: ", portForwardKillErr, "Continuing...")
+				}
+			}()
+
+			time.Sleep(2 * time.Minute)
```
Comment (vishalathreya) [inline test/e2e/fishapp/load/load_stack_test.go:258]: Even though I'm waiting for Fortio components to become active and port-forwarding to run fine, if I comment out this sleep, the POST request inside the Python Load driver script fails with Connection Refused. Works fine after a brief delay/sleep

Comment (bendu) [reply test/e2e/fishapp/load/load_stack_test.go:258]: We currently have no way to coordinate Envoy start and Application start, so this is (sadly) somewhat expected...

File: test/e2e/fishapp/dynamic_stack_test.go [hunk 2]
```diff
@@ -52,6 +52,9 @@ var _ = Describe("test dynamically generated symmetrical mesh", func() {
 		})
 
 		AfterEach(func() {
+			sleep_duration := 6 * time.Hour
```
[the remark this answers is not in the document]
Comment (vishalathreya) [reply test/e2e/fishapp/dynamic_stack_test.go:55]: That can be removed. It was there for testing purposes. Will commit and push that change

**Review by bendu — APPROVED** [review]

**Comment by vishalathreya** [comment]
Modifications to allow e2e/fishapp to create resources and run load tests from config.

**Pull request merged**